packages/mui-system/src/cssVars/cssVarsParser.ts

```diff
@@ -124,9 +124,10 @@ export default function cssVarsParser<T extends Record<string, any>>(
   options?: {
     prefix?: string;
     shouldSkipGeneratingVar?: (objectPathKeys: Array<string>, value: string | number) => boolean;
+    addDefaultValues?: boolean;
   },
 ) {
-  const { prefix, shouldSkipGeneratingVar } = options || {};
+  const { prefix, shouldSkipGeneratingVar, addDefaultValues = false } = options || {};
```

Comment:
Added an additional option to indicate if the default value should be added when generating the CSS variable. This value is set to true in `extendTheme`.